examples/query_lang/claims_alternatives.json

```
@@ -0,0 +1,35 @@
+{
+  "expect_credentials": [
```

```suggestion
  "requested_credentials": [
```
I prefer requested, it sounds more clear and direct than "expect" to me.